### ISSUE TITLE
Cannot copy files, get 'Timeout exceeded'

### ISSUE THREAD

**Issue by jwilf** [mludvig/aws-ssm-tools#2]
This utility is great, but ssm-copy is not working for me:

```
➜  ~ ssm-copy test i-05bxxxxxxxxxxx:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/pexpect/expect.py", line 111, in expect_loop
    incoming = spawn.read_nonblocking(spawn.maxread, timeout)
  File "/usr/local/lib/python3.7/site-packages/pexpect/pty_spawn.py", line 482, in read_nonblocking
    raise TIMEOUT('Timeout exceeded.')
pexpect.exceptions.TIMEOUT: Timeout exceeded.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/ssm-copy", line 204, in <module>
    main()
  File "/usr/local/bin/ssm-copy", line 186, in main
    sender = SsmFileSender(instance_id, profile = args.profile, region = args.region, logger_name = logger_name)
  File "/usr/local/lib/python3.7/site-packages/ssm_tools/talker.py", line 9, in __init__
    self.connect(instance_id, profile, region)
  File "/usr/local/lib/python3.7/site-packages/ssm_tools/talker.py", line 23, in connect
    self.wait_for_prompt()
  File "/usr/local/lib/python3.7/site-packages/ssm_tools/talker.py", line 49, in wait_for_prompt
    self._child.expect('sh.*\$ $')
  File "/usr/local/lib/python3.7/site-packages/pexpect/spawnbase.py", line 341, in expect
    timeout, searchwindowsize, async_)
  File "/usr/local/lib/python3.7/site-packages/pexpect/spawnbase.py", line 369, in expect_list
    return exp.expect_loop(timeout)
  File "/usr/local/lib/python3.7/site-packages/pexpect/expect.py", line 119, in expect_loop
    return self.timeout(e)
  File "/usr/local/lib/python3.7/site-packages/pexpect/expect.py", line 82, in timeout
    raise TIMEOUT(msg)
pexpect.exceptions.TIMEOUT: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x10952a390>
command: /usr/local/bin/aws
args: [b'/usr/local/bin/aws', b'ssm', b'start-session', b'--target', b'i-05b7c0ffb1bc9664a']
buffer (last 100 chars): '\r\nStarting session with SessionId: 1569835485940510000-0723a5e801a6f63f0\r\n$ '
before (last 100 chars): '\r\nStarting session with SessionId: 1569835485940510000-0723a5e801a6f63f0\r\n$ '
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 35112
child_fd: 7
closed: False
timeout: 10
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_re:
    0: re.compile('sh.*\\$ $')
```

ssm-session to the same instance works fine.

The instance is Debian but I'm on Mac OS locally. Installed aws-ssm-tools using pip3. My python installation is:

`#!/usr/local/opt/python/bin/python3.7`

aws-cli and session-manager-plugin versions:

```
aws --version
aws-cli/1.16.248 Python/3.7.3 Darwin/18.7.0 botocore/1.12.238

session-manager-plugin --version
1.1.33.0
```

Thanks

**Comment by mludvig** [comment]
Hi, sorry for a very late reply. `ssm-copy` never quite worked reliably. It's now been superseded by `rsync` + `ssm-ssh` that's much faster and way more reliable. Sorry for that.
